Source/Libraries/openXDA.Model/Emails/EmailTypeBase.cs

```diff
@@ -29,13 +30,15 @@ public abstract class EmailTypeBase
     {
         [PrimaryKey(true)]
         public int ID { get; set; }
-
+        [ParentKey(typeof(EmailCategory))]
+        public int EmailCategoryID { get; set; }
+        [DefaultValue(false)]
+        public bool RequireApproval { get; set; }
+        public bool ShowSubscription { get; set; }
```

Comment:
In C#, bool values default to `false`. So if one of these needs a `DefaultValue` attribute, it would be `ShowSubscription`.
```suggestion
        [DefaultValue(false)]
        public bool RequireApproval { get; set; }
        [DefaultValue(true)]
        public bool ShowSubscription { get; set; }
```